### PR TITLE
Minor edits and clarification

### PR DESCRIPTION
**Pull request by kan-o-ash**

### CODE REVIEW

File: tag-input.md [hunk 1]
```diff
@@ -8,33 +8,37 @@ Write a react component which will serve as an input for the spices used in cook
...
 ## Hints
 
-1. Things do not have to look identical to the example image. What we care about the most is functionality, so focus on that before how the input looks.
-2. Write quality, readable code that you would feel comfortable with deploying to production. **If you are chosen to continue with the interview process, you will have to modify the code you write for the next step of the interview!**
-3. This type of component exists all over the internet, but we want to see you write your own. Third party libraries are OK to use, unless they completely trivialize the project. Use your best judgement, and reach out to your interviewer if you are unsure. We really just want to avoid having the entire project be completed with `npm install react-tag-autocomplete`.
+1. We recommend you prioritize in the following order:
+   - Finish base requirements
+   - Ensure high code quality
+   - Bonus points / style
+1. Focus on functionality before style - things do not need to look identical to the example image.
```
Comment (mako-taco) [inline tag-input.md:39]: 1.
1.
1. 
?

File: tag-input.md [hunk 2]
```diff
@@ -8,33 +8,37 @@ Write a react component which will serve as an input for the spices used in cook
...
+1. Write quality, readable code that you would feel comfortable with deploying to production. **To continue with the interview process, you will have to modify the code you write for the next step of the interview.**
+1. This type of component exists all over the internet, but we want to see you write your own. Third party libraries are OK to use, unless they completely trivialize the project (you should implement the core requirements yourself). Use your best judgement, and reach out to your interviewer if you are unsure.
```
Comment (mako-taco) [inline tag-input.md:41]:
```suggestion
1. Focus on functionality before style - things do not need to look identical to the example image.
2. Write quality, readable code that you would feel comfortable with deploying to production. **To continue with the interview process, you will have to modify the code you write for the next step of the interview.**
3. This type of component exists all over the internet, but we want to see you write your own. Third party libraries are OK to use, unless they completely trivialize the project (you should implement the core requirements yourself). Use your best judgement, and reach out to your interviewer if you are unsure.
```

**Review by mako-taco — APPROVED** [review]

File: tag-input.md (hunk 1 above)
Comment (kan-o-ash) [reply tag-input.md:39]: In markdown you can write a numeric list as
`1.`
`1.`
`1.`
`1.`
Etc and it gets translated to the real numbers on display. So you never have to update numbers if you add additional steps / remove steps etc

Comment (kan-o-ash) [reply tag-input.md:39]: see how it gets rendered:
https://github.com/shefhq/take-home-interview/blob/9d6fdb03ac53b69607eb351bf367192aef269aee/tag-input.md#hints

Comment (kan-o-ash) [reply tag-input.md:39]: see how it gets rendered:
https://github.com/shefhq/take-home-interview/blob/9d6fdb03ac53b69607eb351bf367192aef269aee/tag-input.md#hints

**Pull request merged**